config.json

```diff
@@ -721,7 +727,7 @@
         "name": "Luhn",
         "uuid": "28872cc9-f1ef-487f-9a79-6bf7983148bf",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": ["numbers", "conditionals", "array-transformations"],
```

Comment:
Here is my solution : https://exercism.org/tracks/javascript/exercises/luhn/solutions/TomPradat

I haven't added "array-analysis" despite the fact that I personally use `Array#some` as it does not seem necessary (as we can see in other community solutions)